pylinac/core/profile.py

```diff
@@ -159,6 +160,31 @@ def values(self, value):
             raise TypeError("Values must be a numpy array")
         self._values = value.astype(float)
 
+    @property
+    def dpmm(self) -> NumberLike:
+        """The Dots-per-mm of the profile, defined at isocenter. E.g. if an EPID image is taken at 150cm SID,
+        the dpmm will scale back to 100cm. Added by ACC 3/12/2020"""
```

Comment:
Generally speaking, you don't need to add your name or when this was added as this is authoritatively kept by git and accessible via git blame.

Reply:
My apologies, old habits die hard:)